cloudformation/dynamoDb.yaml

```diff
@@ -240,6 +240,32 @@ Resources:
           - latest
     DeletionPolicy: Retain
 
+  CampaignLocationAnalyticsLatestDBTable:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      AttributeDefinitions:
+      - AttributeName: territory
+        AttributeType: S
+      - AttributeName: campaignId
+        AttributeType: S
+      KeySchema:
+      - AttributeName: territory
+        KeyType: HASH
+      - AttributeName: campaignId
```

Comment:
Which way around is better, is this going to be displayed inside a particular campaign? If so maybe the opposite might be better?

Reply:
@janua I thought about this for a bit. I think this way as Campaign Central will be contextual to the territory which the user will select at the top level of Campaign Central. So this allows you to query for all campaigns in a given territory.

Reply:
Ah I see... Then you can batch get the campaigns!

Reply:
Yeah :)